src/style.css

```diff
@@ -10,12 +10,15 @@ body {
   font-family: 'Courier New', Courier, monospace;
   padding: 50px;
   text-align: center;
+  background-color: #111;
+  color: #fff;
 }
 
```

Comment:
- [ ] OPTIONAL
The title _Leaderboard_ should be aligned to the left of the page, as can be seen in the original design. Kindly ensure to improve upon this aspect.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/151005757-45ee1961-00da-45e9-84f1-cf5e63b2ee20.png)

**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/151006015-b4dc8d59-77a5-4afc-8d6d-3581c5e0067c.png)